tests_integration/nativeapp/test_feature_metrics.py

```diff
@@ -103,7 +103,7 @@ def test_sql_templating_emits_counter(
                 CLICounterField.TEMPLATES_PROCESSOR: 0,
                 CLICounterField.PDF_TEMPLATES: 1,
                 CLICounterField.POST_DEPLOY_SCRIPTS: 1,
-                CLICounterField.PACKAGE_SCRIPTS: 0,
+                CLICounterField.PACKAGE_SCRIPTS: 1,
```

Comment:
test files have package scripts, so this should be 1.

Reply:
Do we want to make the distinction that since the pdf is v1, then POST_DEPLOY_SCRIPTS should count as 0 then?

Reply:
i think it's something we can distinguish in the graphs themselves (like exclude package_scripts = 1). I do not want to touch the post_deploy_scripts part because the goal is to have the bulk part of the code operate in pdf v2 mode, and would make sense to keep the counters for this one with the code itself.